### PR TITLE
Add web page publishing plugin

### PR DESCRIPTION
**Pull request by mekya**
https://github.com/ant-media/Ant-Media-Server/issues/4426

### CODE REVIEW

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

**Comment by mekya** [comment]
Let's make the REST methods comply with standards
https://github.com/WhiteHouse/api-standards

@mustafaboleken

File: WebpageRecordingPlugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+		</dependency>
+		<dependency>
+			<groupId>org.seleniumhq.selenium</groupId>
+			<artifactId>selenium-chrome-driver</artifactId>
+			<version>4.8.1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.seleniumhq.selenium</groupId>
+			<artifactId>selenium-java</artifactId>
+			<version>4.8.1</version>
```
[the remark this answers is not in the document]
Comment (mustafaboleken) [reply WebpageRecordingPlugin/pom.xml:308]: @sonatype-lift ignoreall

**Comment by mustafaboleken** [comment]
> Let's make the REST methods comply with standards https://github.com/WhiteHouse/api-standards
> 
> @mustafaboleken

It's ready to review

**Review by lastpeony — APPROVED** [review]

**Pull request merged**